ipaserver/install/service.py

```diff
@@ -347,7 +347,10 @@ def _ldap_mod(self, ldif, sub_dict=None, raise_on_err=True,
                   ldap_uri=None, dm_password=None):
         pw_name = None
         fd = None
-        path = os.path.join(paths.USR_SHARE_IPA_DIR, ldif)
+        if not os.path.isabs(ldif):
+            path = os.path.join(paths.USR_SHARE_IPA_DIR, ldif)
+        else:
+            path = ldif
```

Comment:
This check is not needed:

> https://docs.python.org/3/library/os.path.html#os.path.join
> If a component is an absolute path, all previous components are thrown away and joining continues from the absolute path component.

Reply:
The check is there because the existing code always adds /usr/share/ipa to the path.